replication/source.go

```diff
@@ -121,11 +122,12 @@ func (d *downloader) download(seq int, ext string) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode == 404 {
+		log.Print("Remote file does not exist ", url)
```

Comment:
Never log _and_ return an error. You can log in `downloadTillSuccess` where the error is handled.

Reply:
Good point!
The issue is that the `url` is not available where we catch the error.
Should I make the error itself aware of the URL so we can use it later on the stack?

Reply:
Yes, you can use github.com/pkg/errors.New (like here https://github.com/omniscale/imposm3/blob/master/database/database.go#L84)

Reply:
Done through https://github.com/omniscale/imposm3/pull/158/commits/25f165d74ee27b1038dc6f9d5053dc4f1868e183